.jenkins/infrastructure/docker/build_mystikos_base_docker.Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+pipeline {
+    agent {
+        label 'ACC-1804-DC4'
+    }
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "MYST_VERSION", description: "Mystikos release version (Example: 0.5.0). See https://github.com/deislabs/mystikos/releases for release versions")
+        string(name: "REPOSITORY_NAME", defaultValue: "deislabs/mystikos", description: "GitHub repository to checkout")
+        string(name: "BRANCH_NAME", defaultValue: "master", description: "The branch used to checkout the repository")
+        string(name: "MYST_BASE_CONTAINER_TAG", defaultValue: "latest", description: "The tag for the new Mystikos base Docker container.")
```

Comment:
Nit: Do we want to have the tag default to latest, when there is already `TAG_LATEST` ?

Reply:
Good question. I guess if we assume that the `latest` tag is of some importance, we will want to have a different default tag. On the other hand, if `latest` just signifies the latest build (no guarantees on stability/correctness of the container), then we could get rid of the `TAG_LATEST` parameter. Thoughts?

Reply:
Sorry, let me clarify that this is just an inefficiency where the container is built with the `latest` tag (if left with default value) regardless of whether `TAG_LATEST` is true or false.

When pushing to a repository it should be assumed that the container has been reasonably tested and stable, and `latest` just refers to the most recent build.